### ISSUE TITLE
osquery doesn't seem to respect --logger_min_stderr or --logger_min_status when --logger_plugin set

### ISSUE THREAD

**Issue by fmanco** [osquery/osquery#4608]
### What version of `osquery` are you using? What operating system. Be specific.

```
osquery_version = 3.2.8
    os_platform = rhel
```

### What have you already tried?

Start osqueryd with
```
osqueryd --logger_plugin=<my_plugin> --logtostderr --logger_min_stderr=2
```
### What did you expect to see?

Should only see `ERROR` level status logs on the console.

### What did you see instead?

Seeing status logs of all levels on the console.

**Comment by guliashvili** [comment]
If logtostderr or alsologtostderr (both are GLOG flags not the osquery flags) are set to true, severity of the log is not considered while printing things to stderr.

However you confusion looks more than understandable. (And even if you write --logtostderr=false things will not work)

1. In command line flags docs osquery and glog flags are mixed.
2. Default values of flag(logtostderr which is GLOG flag and osquery just modifies it) does not match to the default value in the GLOG docs(because of the way osquery modifies it)
3. Code is hard to follow and flags are too tangled to each other. 
For example,
    if you write disable_logging=true logtostderr is automatically set to true. 
    If you are logging from external process logtostderr is again enabled.
    **If logger plugin is not named filesystem logtostderr is automatically enabled https://github.com/facebook/osquery/blob/master/osquery/logger/logger.cpp#L285**


 I will need some time to untangle the code. Reduce number of flags and exactly say what kind of behavior is being changed.

Will assign the issue to myself if you don't mind @fmanco
